winrar

```diff
@@ -0,0 +1,23 @@
+{% set versions = {'5':['61']} %}
+
+winrar:
+{% for major, subversions in versions.items() %}
+{% for minor in subversions %}
+  '{{major}}.{{minor}}.0':
+    {% if grains['cpuarch'] == 'AMD64' %}
+    full_name: 'WinRAR {{major}}.{{minor}} (64-bit)'
+    installer: 'https://www.netzmechanik.de/dl/4/winrar-x64-{{major}}{{minor}}.exe'
```

Comment:
Let's use rarlab's official url. I don't know what this `netzmechanik.de` is.

https://www.rarlab.com/rar/winrar-x64-561.exe

So, `https://www.rarlab.com/rar/winrar-x64-{{major}}{{minor}}.exe`

Reply:
The link come from the website https://www.winrar.de/
In germany this page is listed first in the google search, but you are right, I will change the URL.